src-js/webpack/src/mappings/reducers/edit-mapping-reducers.js

```diff
@@ -97,10 +97,12 @@ export const RuleGroupReducer = createReducer(null, {
   [MAPPING_TERMS_CHANGED]: (state, action) => {
     const taxonomy = action.payload.taxonomy;
     const terms = action.payload.terms;
+
     // Check if there is no term items for the taxonomy, to prevent duplication.
     if (0 === state.ruleFieldTwoOptions.filter(e => e.taxonomy === taxonomy).length) {
```

Comment:
there was an or condition in this line in the last commit, please add it and rebuild the files with `yarn build `

Reply:
` || taxonomy === 'post_taxonomy'`